dired-filter.el

```diff
@@ -393,6 +393,20 @@ See `dired-filter-stack' for the format of FILTER-STACK."
   "The face used to highlight pair overlays."
   :group 'dired-filter-group)
 
+(defvar dired-filter-group-imenu-generic-expression '("Group" "^ *\\[\\(.*\\)\\]" 1)
+  "An alist of menus for accessing locations in documents with imenu
+
+It is a list of lists of the form
+(MENU-TITLE REGEXP INDEX [FUNCTION] [ARGUMENTS...]
+
+See `imenu-generic-expression' for more information.
+)
+")
+
+(add-hook 'dired-filter-group-mode-hook
```

Comment:
This should probably only happen when the mode is activated, and then the hook should be removed when the mode is deactivated.  Which would probably require the hook function to be defined instead of a lambda.  Does this make sense?

Reply:
Yes it does, thanks for having I look, I'll keep pushing